conda-recipe/bld.bat

```diff
@@ -65,5 +64,5 @@ for /f %%f in ('dir /b /S .\dist') do (
 :: Copy wheel package
 if NOT "%WHEELS_OUTPUT_FOLDER%"=="" (
     copy dist\dpctl*.whl %WHEELS_OUTPUT_FOLDER%
-    if errorlevel 1 exit 1
+    if %ERRORLEVEL% neq 0 exit 1
```

Comment:
What is about cmake file copying above? Should we use `%ERRORLEVEL% neq 0` there also?

Reply:
And probably we should update `conda-recipe/run_test.bat` and `libsyclinterface/dbg_build.bat` to use the same generic check everywhere.